what

```diff
@@ -1,10 +1,15 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 
 # Like "w", but finds all processes associated with a TTY (not just
 # those registered in wtmp), and reports all users that are running
 # anything.  In particular, unlike "w", this will shows things running
 # in detached screens/tmuxen.
 
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
```

Comment:
I think you only need to import `print_function`, since these other features are not used here.